src/features/projects/components/projectDetails/ProjectInfo.tsx

```diff
@@ -244,20 +244,10 @@ function ProjectInfo({ project }: Props): ReactElement {
             <div className={styles.infoTitle}>
               {t('manageProjects:siteOwnership')}
             </div>
-            {project.metadata?.siteOwnerType && (
-              <div className={styles.infoText} style={{ fontWeight: 'bold' }}>
-                {ownerTypes.map((ownerType: any, index: any) => {
-                  return (
-                    <React.Fragment key={ownerType}>
-                      {t(`manageProjects:${ownerType}`)}
-                      {index === ownerTypes.length - 2
-                        ? ' and '
-                        : index === ownerTypes.length - 1
-                        ? '.'
-                        : ', '}
-                    </React.Fragment>
-                  );
-                })}
+
+            {project?.metadata?.landOwnershipType && (
+              <div className={styles.infoText}>
+                {project?.metadata?.landOwnershipType}
```

Comment:
This new fields require translations for all possible keys:

<img width="363" alt="Bildschirmfoto 2022-06-04 um 11 23 32" src="https://user-images.githubusercontent.com/1532418/171993288-a96042f5-2d98-4605-9827-2d94268c8fdd.png">

Reply:
@norbertschuler This is handled.